src/plugins/segmentio/index.ts

```diff
@@ -12,7 +12,7 @@ import standard from './fetch-dispatcher'
 import { normalize } from './normalize'
 import { scheduleFlush } from './schedule-flush'
 
-export interface SegmentioSettings {
+export type SegmentioSettings = {
```

Comment:
Why the change?

Reply:
TypeScript was complaining that `SegmentioSettings` wasn't a subset of the `JSONObject` shape (but it is since it's just an interface containing properties) - changing this from an interface to a type allowed TypeScript to recognize that.